### ISSUE TITLE
Enforcing null in tobit

### ISSUE THREAD

**Issue by lboxell** [droodman/boottest#1]
Hi @droodman,

I've been trying to use `boottest` with a tobit model. The `boottest` help file notes that tobit models did not work up until Stata 15. However, there still appears to be issues when using Stata 15 and trying to enforce the null hypothesis.

Minimum working example is below:

```
version
sysuse auto, clear
replace price = 6000 if price < 6000

tobit price mpg, ll vce(cluster rep78)
boottest mpg, reps(1000)   
boottest mpg, nonull reps(1000)   
```

My output:

``` 
. version
version 15.1

. sysuse auto, clear
(1978 Automobile Data)

. replace price = 6000 if price < 6000
(51 real changes made)

. 
. tobit price mpg, ll vce(cluster rep78)

Refining starting values:

Grid node 0:   log likelihood =  -261.1434

Fitting full model:

Iteration 0:   log pseudolikelihood =  -261.1434  
Iteration 1:   log pseudolikelihood =  -234.4658  
Iteration 2:   log pseudolikelihood = -228.21312  
Iteration 3:   log pseudolikelihood = -226.69429  
Iteration 4:   log pseudolikelihood = -226.63129  
Iteration 5:   log pseudolikelihood = -226.63087  
Iteration 6:   log pseudolikelihood = -226.63087  

Tobit regression                                Number of obs     =         69
                                                   Uncensored     =         21
Limits: lower = 6,000                              Left-censored  =         48
        upper = +inf                               Right-censored =          0

                                                F(   1,     68)   =       2.69
                                                Prob > F          =     0.1054
Log pseudolikelihood = -226.63087               Pseudo R2         =     0.1181

                                  (Std. Err. adjusted for 5 clusters in rep78)
------------------------------------------------------------------------------
             |               Robust
       price |      Coef.   Std. Err.      t    P>|t|     [95% Conf. Interval]
-------------+----------------------------------------------------------------
         mpg |  -668.2116   407.1391    -1.64   0.105    -1480.645    144.2218
       _cons |   17079.71   6727.683     2.54   0.013     3654.824    30504.59
-------------+----------------------------------------------------------------
 var(e.price)|   2.19e+07    6803213                      1.18e+07    4.07e+07
------------------------------------------------------------------------------

. boottest mpg, reps(1000)   

Re-running regression with null imposed.

option constraints() not allowed
Error imposing null. Perhaps tobit does not accept the constraints(), from(), and iterate() options, 
> as needed.
r(198);

. boottest mpg, nonull reps(1000)   

Warning: with 5 Clusters, the number of replications, 1000, exceeds the universe of Rademacher draws,
>  2^5 = 32. Sampling each once. 
Consider Webb weights instead, using weight(webb).

Score bootstrap, null not imposed, 32 replications, Wald test, bootstrap clustering by rep78, Rademac
> her weights:
  mpg

                                        t(4) =    -1.6169
                                    Prob>|t| =     0.3750

. 
```

**Comment by droodman** [comment]
Thanks for bringing this to my attention. I believe I've just fixed it in release 2.3.7. I'll have this posted on SSC. I suggest you install it by typing in Stata:
`net from https://raw.github.com/droodman/boottest/v2.3.7`
`net install boottest, replace`

**Comment by lboxell** [comment]
Great, thanks!
